pipeline/batch.go

```diff
@@ -178,6 +182,25 @@ func (n *QueryNode) ChainMethods() map[string]reflect.Value {
 //
 // It is recommended to use QueryNode.Align and QueryNode.Offset in conjunction with
 // group by time dimensions so that the time bounds match up with the group by intervals.
+// To automatically align the group by intervals to the start of the query time,
+// use QueryNode.AlignGroup. This is useful in more complex situations, such as when
+// the groupBy time period is longer than the query frequency.
+//
+// Example:
+//    batch
+//        |query(...)
+//            .period(5m)
+//            .every(30s)
+//            .groupBy(time(1m), 'tag1', 'tag2')
+//            .align()
+//            .alignGroup()
+//
+// For the above example, without QueryNode.AlignGroup, every other query issued by Kapacitor
+// (at :30 past the minute) will align to :00 seconds instead of the desired :30 seconds,
+// which would create 6 group by intervals instead of 5, the first and last of which
+// would only have 30 seconds of data instead of a full minute.
+// If QueryNode.Offset is used in conjunction with QueryNode.AlignGroup,
```

Comment:
The QueryNode.Offset is different from the group by time offset. This should read 

`If the group by time offset (i.e. time(t, offset)) is used in conjunction with QueryNode.AlignGroup,`